src/translate.rs

```diff
@@ -21,7 +21,14 @@
 //! ```
 //!
 //! `ToGlibPtr`, `FromGlibPtrNone`, `FromGlibPtrFull` and `FromGlibPtrBorrow` work on `gpointer`s
-//! and support different modes of ownership transfer.
+//! and ensure correct ownership of references on the Rust side
```

Comment:
Maybe s/references/data/

Reply:
I used "value" later. Is "value" or "data" better?

Reply:
IMHO "value" is better.